addons/web/static/src/views/fields/properties/property_value.scss

```diff
@@ -26,6 +26,14 @@ input.o_field_property_input[type=number] {
     }
 }
 
+.o_property_field_value {
```

Comment:
Unless I'm mistaken, this class is already defined at line 66, could add your rules there instead of duplicating?